### DISCUSSION TITLE
Should fido2-token list platform authenticators (macOS TouchID)?

### DISCUSSION THREAD

**Discussion by restena-sw** [Yubico/libfido2#649] [Q&A] [answered]
I have libfido2 on macOS 13 and executed

`fido2-token -L`

This resulted in an empty list. Once I plug in a roaming authenticator (Yubikey 5C) that roaming authenticator is found and displayed. So the tool is working, to some extent.

Is it conceptually not possible, or intentionally omitted, or a bug in fido2-token that the platform authenticator on the MacBook Pro is not listed?

**Comment by LDVG** [comment, marked as answer]
libfido2 currently only implements a HID backend for macOS, i.e. only roaming authenticators will be found.
